app/models/concerns/searchable.rb

```diff
@@ -0,0 +1,67 @@
+module Searchable
+  extend ActiveSupport::Concern
```

Comment:
I think search functions benefit from being clustered together. Their form is more similar to each other than to the different objects they are querying, so when you're going in to add any new search capability having the other examples at hand seems helpful.